vtex/utils/types.ts

```diff
@@ -739,7 +739,7 @@ export interface Item {
   unitMultiplier: number;
   modalType: unknown | null;
   images: Image[];
-  Videos: string[];
+  videos: string[];
```

Comment:
Just to check: are you sure about this type change? it's possibly a breaking change if anyone was using or it ever existed

Reply:
I'm not sure if anyone would be using it because the API doesn't return any "Videos" (with a capital V), only "videos" (with a lowercase v). Consequently, it wasn't even possible to do anything with this object.